src/inmanta/server/services/compilerservice.py

```diff
@@ -728,7 +728,8 @@ async def _auto_recompile_wait(self, compile: data.Compile) -> None:
         else:
             env = await data.Environment.get_by_id(compile.environment)
             wait_time = await env.get(data.RECOMPILE_BACKOFF)
-            LOGGER.info("The recompile_backoff environment setting is enabled and set to %s seconds.", wait_time)
+            if wait_time:
+                LOGGER.info("The recompile_backoff environment setting is enabled and set to %s seconds.", wait_time)
```

Comment:
I would not log less then what use used to log. I would just make the message more correct:

```suggestion
            if wait_time:
                LOGGER.info("The recompile_backoff environment setting is enabled and set to %s seconds.", wait_time)
            else:
                LOGGER.info("The recompile_backoff environment setting is disabled)
```